src/gui/src/settings/options-window.h

```diff
@@ -116,6 +123,7 @@ class OptionsWindow : public QDialog
 		QList<QLineEdit*> m_customNames, m_customTags, m_filenamesConditions, m_filenamesFilenames, m_filenamesFolders;
 		QList<TokenSettingsWidget*> m_tokenSettings;
 		QList<QPair<QLineEdit*, QLineEdit*>> m_metadataPropsys, m_metadataExiftool;
+		QList<std::pair<QCheckBox*, QSpinBox*>> csPairs;
```

Comment:
While I don't mind STD containers, I don't really recommend mixing both 😅

Reply:
Well, I would have preferred to use an std::list but I didn't want to add the header when we already have this available. I will try changing to QPair and see if that works.

Reply:
I already did 😅 They have the same API so it didn't really change anything